pages/treeherder.py

```diff
@@ -233,6 +237,19 @@ def find_expanded_group_content(self):
         def jobs(self):
             return [self.Job(self.page, root=el) for el in self.find_elements(*self._jobs_locator)]
 
+        @property
+        def results_visible(self):
+            return self.find_element(*self._resultset_visible_locator)
+
+        @property
+        def runnable_jobs(self):
+            return [self.Job(self.page, root=el) for el in self.find_elements(*self._runnable_jobs_locator)]
+
+        def add_new_jobs(self):
```

Comment:
This is not used, and therefore neither is `runnable_jobs` or `_runnable_jobs_locator`.